phpcs.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <ruleset name="DataValuesJavaScript">
-	<rule ref="vendor/wikibase/wikibase-codesniffer/Wikibase">
-		<exclude name="Generic.Arrays.DisallowLongArraySyntax" />
-	</rule>
+	<rule ref="./vendor/wikibase/wikibase-codesniffer/Wikibase" />
```

Comment:
To be honest I don't know. But I have seen Umherirrender adding this `./` to all phpcs.xml files, and I believe being consistent is important.